.github/workflows/staging.yaml

```diff
@@ -1,15 +1,13 @@
-name: Staging Deployment to Bots Infrastructure
+name: Staging Deployment to zinfra.io
```

Comment:
zinfra.io is a domain name, not a name of an environment, and this currently doesn't deploy in any other places than before, or does it?

Reply:
We have Bot environments that we call Annayotto (Production) and Dagobah (staging). I need some new name to refer to Wire's staging (EKS). How do you want to call it? Everybody within the team calls is `zinfra`

Reply:
If this deploys to Dagobah or Annayotto, it should be called as this. If this deploys to `prod` and `staging` EKS, it should be called as such.

However, from looking at the pipeline, it doesn't seem to be deploying anywhere - just building and tagging containers? :thinking: